pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -29,6 +29,7 @@
 import torch.distributed
 from torch.nn.parallel.distributed import DistributedDataParallel
 
+import pytorch_lightning as pl
```

Comment:
n00b question: does this introduce a recursive import?

Reply:
No, we only have the trainer and the LM imported there. this is relevant for docs-compatible typing, since spinx cannot really deal with forward declarations.